docs/src/components/landing/DemoVideo.js

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import { styled } from '@mui/material/styles';
+
+const Video = styled('video')(({ theme }) => ({
+  borderStyle: 'solid',
+  borderColor:
+    theme.palette.mode === 'dark' ? theme.palette.primaryDark[900] : theme.palette.grey[100],
+  borderRadius: '10px 10px 0 0 ',
+  overflow: 'hidden',
+  objectFit: 'cover',
+  objectPosition: 'top',
+  outline: `1px solid ${
+    theme.palette.mode === 'dark' ? theme.palette.primaryDark[400] : theme.palette.grey[300]
+  }`,
+}));
+
+export default function DemoVideo() {
+  return (
+    <React.Fragment>
+      <Video
+        sx={{
+          height: { lg: 655 },
```

Comment:
This leads to a truncated video:

<img width="1171" alt="Screenshot 2022-08-25 at 12 54 10" src="https://user-images.githubusercontent.com/3165635/186646461-9a0f54a2-dbf0-417a-9f96-2b3a8c07d7bf.png">

Without it:

```suggestion
```

<img width="1175" alt="Screenshot 2022-08-25 at 12 54 36" src="https://user-images.githubusercontent.com/3165635/186646536-58dc953a-906e-422e-8e63-994c48fdcb63.png">

Reply:
If it was added to avoid layout shift, I think that we could use this trick: https://stackoverflow.com/questions/600743/how-to-get-div-height-to-auto-adjust-to-background-size.

Reply:
Without the `height`, the video gets vertically truncated; I've tried to add a new approach with specific heights for all breakpoints which feels like it works the best across all viewport sizes